packages/terra-arrange/tests/nightwatch/AlignedAllLongWords.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Arrange from '../../lib/Arrange';
+import { ArrangeWrapper, longWordText } from './examplesetup';
+
+const AlignedAllArrange = () => (
+  <div>
+    <ArrangeWrapper >
+      <Arrange
+        id="default"
+        fitStart={longWordText}
+        fill={longWordText}
+        fitEnd={longWordText}
+        fitStartAttributes={{ className: 'test1', style: { maxWidth: '10px', wordWrap: 'break-word' } }}
+        fillAttributes={{ className: 'test2', style: { maxWidth: '20px', wordWrap: 'break-word' } }}
+        fitEndAttributes={{ className: 'test3', style: { maxWidth: '30px', wordWrap: 'break-word' } }}
+      />
+    </ArrangeWrapper>
+
+    <ArrangeWrapper >
+      <Arrange
+        id="center"
+        align="center"
+        fitStart={longWordText}
+        fill={longWordText}
+        fitEnd={longWordText}
+        fitStartAttributes={{ className: 'test1', style: { maxWidth: '10px', wordWrap: 'break-word' } }}
+        fillAttributes={{ className: 'test2', style: { maxWidth: '20px', wordWrap: 'break-word' } }}
+        fitEndAttributes={{ className: 'test3', style: { maxWidth: '30px', wordWrap: 'break-word' } }}
+      />
+    </ArrangeWrapper>
+
+    <ArrangeWrapper >
+      <Arrange
+        id="bottom"
+        align="bottom"
+        fitStart={longWordText}
+        fill={longWordText}
+        fitEnd={longWordText}
+        fitStartAttributes={{ className: 'test1', style: { maxWidth: '10px', wordWrap: 'break-word' } }}
+        fillAttributes={{ className: 'test2', style: { maxWidth: '20px', wordWrap: 'break-word' } }}
+        fitEndAttributes={{ className: 'test3', style: { maxWidth: '30px', wordWrap: 'break-word' } }}
+      />
+    </ArrangeWrapper>
+
+    <ArrangeWrapper >
+      <Arrange
+        id="stretch"
+        align="stretch"
+        fitStart={longWordText}
+        fill={longWordText}
+        fitEnd={longWordText}
+        fitStartAttributes={{ className: 'test1', style: { maxWidth: '10px', wordWrap: 'break-word' } }}
```

Comment:
When I initially looked at this test page, the examples looked broken, took me a bit to make sense of what I was looking at. 
<img width="1200" alt="screen shot 2017-10-20 at 6 03 11 am" src="https://user-images.githubusercontent.com/633148/31818206-bc58426c-b55c-11e7-946a-4b82c6421e73.png">

I realized you went with smaller max-widths to likely test the word-wrap behavior. I'd add in some longer words, and increase the max-width on these examples to more realistic widths. The words can be whatever crazy long combinations to test wrapping.